src/main/java/com/target/ready/library/system/Service/LibrarySystemService.java

```diff
@@ -26,12 +26,19 @@ public class LibrarySystemService {
     public LibrarySystemService(WebClient webClient) {
         this.webclient = webClient;
     }
-    public List<Book> getAllBooks(){
-        List<Book> book_list= webclient.get().uri("http://localhost:8080/library_service_api/v1/allBooks").accept(MediaType.APPLICATION_JSON)
+    public List<Book> getAllBooks(int pageNumber, int pageSize){
+        List<Book> book_list= WebClient.builder()
+                .baseUrl("http://localhost:8080/library_service_api/v1")
```

Comment:
Please use config files for configs and avoid hardcoding.

Reply:
okay will do that